tools/codegen/core/gen_upb_api.sh

```diff
@@ -33,125 +33,6 @@ else
-  "envoy/config/endpoint/v3/endpoint.proto" \
-  "envoy/config/endpoint/v3/load_report.proto" \
-  "envoy/config/listener/v3/api_listener.proto" \
-  "envoy/config/listener/v3/listener_components.proto" \
-  "envoy/config/listener/v3/listener.proto" \
-  "envoy/config/listener/v3/quic_config.proto" \
-  "envoy/config/listener/v3/udp_listener_config.proto" \
-  "envoy/config/metrics/v3/stats.proto" \
-  "envoy/config/overload/v3/overload.proto" \
-  "envoy/config/rbac/v3/rbac.proto" \
-  "envoy/config/route/v3/route_components.proto" \
-  "envoy/config/route/v3/route.proto" \
-  "envoy/config/route/v3/scoped_route.proto" \
-  "envoy/config/trace/v3/http_tracer.proto" \
-  "envoy/extensions/clusters/aggregate/v3/cluster.proto" \
-  "envoy/extensions/filters/common/fault/v3/fault.proto" \
-  "envoy/extensions/filters/http/fault/v3/fault.proto" \
-  "envoy/extensions/filters/http/rbac/v3/rbac.proto" \
-  "envoy/extensions/filters/http/router/v3/router.proto" \
-  "envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto" \
-  "envoy/extensions/transport_sockets/tls/v3/cert.proto" \
-  "envoy/extensions/transport_sockets/tls/v3/common.proto" \
-  "envoy/extensions/transport_sockets/tls/v3/secret.proto" \
-  "envoy/extensions/transport_sockets/tls/v3/tls.proto" \
-  "envoy/service/cluster/v3/cds.proto" \
-  "envoy/service/discovery/v3/ads.proto" \
-  "envoy/service/discovery/v3/discovery.proto" \
-  "envoy/service/endpoint/v3/eds.proto" \
-  "envoy/service/listener/v3/lds.proto" \
-  "envoy/service/load_stats/v3/lrs.proto" \
-  "envoy/service/route/v3/rds.proto" \
-  "envoy/service/route/v3/srds.proto" \
-  "envoy/service/status/v3/csds.proto" \
-  "envoy/type/http/v3/path_transformation.proto" \
-  "envoy/type/matcher/v3/metadata.proto" \
-  "envoy/type/matcher/v3/node.proto" \
-  "envoy/type/matcher/v3/number.proto" \
-  "envoy/type/matcher/v3/path.proto" \
-  "envoy/type/matcher/v3/regex.proto" \
-  "envoy/type/matcher/v3/string.proto" \
-  "envoy/type/matcher/v3/struct.proto" \
-  "envoy/type/matcher/v3/value.proto" \
-  "envoy/type/metadata/v3/metadata.proto" \
-  "envoy/type/tracing/v3/custom_tag.proto" \
-  "envoy/type/v3/http.proto" \
-  "envoy/type/v3/percent.proto" \
-  "envoy/type/v3/range.proto" \
-  "envoy/type/v3/semantic_version.proto" \
-  "xds/annotations/v3/status.proto" \
-  "xds/core/v3/authority.proto" \
-  "xds/core/v3/collection_entry.proto" \
-  "xds/core/v3/context_params.proto" \
-  "xds/core/v3/resource_locator.proto" \
-  "xds/core/v3/resource_name.proto" \
-  "xds/core/v3/resource.proto" \
-  "xds/data/orca/v3/orca_load_report.proto" \
-  "xds/type/v3/typed_struct.proto")
-
-INCLUDE_OPTIONS="-I=$PWD/third_party/xds \
-  -I=$PWD/third_party/envoy-api \
-  -I=$PWD/third_party/googleapis \
-  -I=$PWD/third_party/protobuf/src \
-  -I=$PWD/third_party/protoc-gen-validate \
-  -I=$PWD"
-
-for i in "${proto_files[@]}"
-do
-  echo "Compiling: ${i}"
-  $PROTOC \
```

Comment:
I think I'm missing something.  With this code removed, what is generating the files in src/core/ext/upb-generated and src/core/ext/upbdefs-generated?

Reply:
`tools/codegen/core/gen_upb_api_from_bazel_xml.py` will generate those files from BUILD like Makefile is generated from BUILD.